doc/installation_osx.md

```diff
@@ -1,14 +1,14 @@
-# Mac OS Installation  
+# macOS Installation  
 
-**Note:** OSX support for the full range of functionality offered by the SDK is not yet complete. If you need support for R200 or the SR300, [legacy librealsense](https://github.com/IntelRealSense/librealsense/tree/legacy) offers a subset of SDK functionality. 
+**Note:** macOS support for the full range of functionality offered by the SDK is not yet complete. If you need support for R200 or the SR300, [legacy librealsense](https://github.com/IntelRealSense/librealsense/tree/legacy) offers a subset of SDK functionality. 
 
 ## Building from Source
 
 1. Install XCode 6.0+ via the AppStore
 2. Install the Homebrew package manager via terminal - [link](http://brew.sh/)
 3. Install the following packages via brew:
   * `brew install libusb pkg-config`
-  * `brew install homebrew/versions/glfw3`
+  * `brew install glfw3`
```

Comment:
Hello @wspr,
The following line is used in Travis.yml for CI setup. Currently it has a different syntax : `brew install homebrew/core/glfw3;`
Merging this PR as-is would create inconsistency between the two instances.

Can you modify it to `brew install homebrew/core/glfw3;`  make it consistent with the above usage in Travis.yml?
Alternatively, you may extend this PR to also modify Travis.yml to the same format.